src/app/app.html

```diff
@@ -7,11 +7,12 @@
 
   <ion-content>
     <ion-list>
-      <button menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
-        {{p.title}}
+      <p ion-item color="primary"><b>{{_placeService.placeIsSelected()?_placeService.selectedPlace.name:"Aucun foyer sélectionné"}}</b></p>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)